python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -72,10 +80,8 @@ def do_collective_operation(self, keep_going, method, *args, **kwds):
 
     def ids(self, minid=None, maxid=None):
         "Get the list of job ids. 'minid' and 'maxid' specify optional (inclusive) slice range."
-        if maxid is None:
-            maxid = sys.maxsize
-        if minid is None:
-            minid = 0
+        maxid = maxid if maxid not None else sys.maxsize
+        minid = minid if minid not None else 0
```

Comment:
> firstly, the code above is invalid Python

good spot

> means that too many things are happening on one line.

As I said generally I agree but in the above case I have no strong preference as I think to expand such a short ternary makes the code more bulky than perhaps needed and so I feel that the intention or flow of the code is not as easily conveyed or slightly hindered. It comes down to preference as always :wink: